### DISCUSSION TITLE
Textadept books by Mitchell

### DISCUSSION THREAD

**Discussion by sigzero** [orbitalquark/textadept#23] [General]
I just have a couple of questions about the "Textadept Quick Reference" and "Lua Quick Reference".

1. Does the purchase cover updates for the current digital versions only or is it one and done for the current and future versions? Different authors do different things here.
2. How is errata handled?

Thanks!

**Comment by orbitalquark** [comment]
1. Likely one and done. If you purchase in a reasonable time period before a new edition, you'll get a free update.
2. When I am notified of errata, it is listed on the website. Depending on the nature of reported issues, a new digital edition may be pushed out to buyers.

**Comment by sigzero** [comment]
Awesome thanks.
